README.md

```diff
@@ -404,6 +404,18 @@ Sidekiq.configure_server do |config|
 end
 ```
 
+To monitor general Sidekiq stats:
```

Comment:
A note about how these stats are "global" is probably in order, cause if you are collecting on multiple nodes that are working the same queue you will get dupes.

Reply:
Not sure if this is overkill and you'd rather keep the previous format, but in https://github.com/discourse/prometheus_exporter/pull/194/commits/951c98a7918c3cec168126538cb90dd837b0ca50 I've rewritten this section a little bit, with the idea of giving an initial example with everything necessary, some explanations, and mentioning the option of running a long-lived process for `SidekiqQueue` and `SidekiqStats`. 

I think this would help people trying to use this gem. Thank you!